packages/cli/test/integration/auth.api.test.ts

```diff
@@ -1,21 +1,23 @@
 import express = require('express');
+import type { SuperAgentTest } from 'supertest';
 import validator from 'validator';
-
-import config = require('../../config');
-import * as utils from './shared/utils';
-import { LOGGED_OUT_RESPONSE_BODY } from './shared/constants';
+import config from '../../config';
 import { Db } from '../../src';
+import { AUTH_COOKIE_NAME } from '../../src/constants';
 import type { Role } from '../../src/databases/entities/Role';
+import type { User } from '../../src/databases/entities/User';
+import { LOGGED_OUT_RESPONSE_BODY } from './shared/constants';
 import { randomValidPassword } from './shared/random';
 import * as testDb from './shared/testDb';
-import { AUTH_COOKIE_NAME } from '../../src/constants';
+import * as utils from './shared/utils';
 
 jest.mock('../../src/telemetry');
 
 let app: express.Application;
 let testDbName = '';
 let globalOwnerRole: Role;
 let globalMemberRole: Role;
+let authAgent: (user: User) => SuperAgentTest;
```

Comment:
Nit: This typing is frequently repeated - we could consider an `AuthAgent` type.